projects/ng-components-docs/src/app/examples/components/table/table-examples.component.html

```diff
@@ -0,0 +1,37 @@
+<div class="row">
+    <div class="col-12">
+        <label>Table with all features (sort, pagination and select rows)</label>
+        <seb-table
+            [rows]="rows"
+            [headerList]="headerList"
+            [sortInfo]="sortInfo"
+            (sortClicked)="handleSortClicked($event)"
+            [selectable]="true"
+            [isAllSelected]="isAllSelected"
+            [selectedRowIndexes]="selectedRows[curentPage]"
+            (rowClicked)="handleSelectRow($event)"
+            (selectAllClicked)="handleSelectAll()"
+        ></seb-table>
+        <sebng-pagination
+            [value]="curentPage + 1"
+            (change)="handleChangePagination($event)"
+            [size]="data.length"
+            [offset]="maxItemsPerPage"
+        ></sebng-pagination>
+    </div>
+
+    <div class="col-12">
+        <label>Table without sorting</label>
+        <seb-table [rows]="data" [headerList]="headerList"></seb-table>
+    </div>
+
+    <div class="col-12">
+        <label>Table with selectable rows</label>
+        <seb-table [rows]="data" [headerList]="headerList" [selectable]="true"></seb-table>
```

Comment:
will it be better if we include the select all example here so when we click on select all checkbox, all rows will be selected?

Reply:
I will refactor this so that this example uses it's own table service. I can only use one service per component